accounting/accounting/doctype/stock_entry/stock_entry.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2023, Akhil and contributors
+# For license information, please see license.txt
+from datetime import datetime
+
+import frappe
+from accounting.accounting.doctype.stock_entry_table_item.stock_entry_table_item import (
+	StockEntryTableItem,
+)
+from frappe.model.document import Document
+
+
+class StockEntry(Document):
+	def validate_item_metadata(self, item: StockEntryTableItem):
+		if item.quantity < 1:
```

Comment:
It can be decimal :grimacing:

Reply:
Hmm changed it to a float value now, initial implementation had it as integer.